go.mod

```diff
@@ -26,7 +26,7 @@ require (
 	github.com/aws/aws-sdk-go-v2/service/s3 v1.30.3
 	github.com/aws/smithy-go v1.13.5
 	github.com/bmizerany/pat v0.0.0-20160217103242-c068ca2f0aac
-	github.com/canonical/go-dqlite v1.11.5
+	github.com/canonical/go-dqlite v1.11.7
```

Comment:
@SimonRichardson I believe this update allows us to handle this comment.
https://github.com/juju/juju/pull/15331#discussion_r1149861451

We discussed it previously. Should be able to use `gopkg.in/yaml.v3` now.